src/rules/newline-after-import.js

```diff
@@ -76,6 +76,20 @@ module.exports = {
   create(context) {
     let level = 0;
     const requireCalls = [];
+    let options = setOptions(context);
+
+    function setOptions(context) {
+      if (context.options[0]) {
+        if ('count' in context.options[0] && 'considerComments' in context.options[0] ) {
+          return context.options[0]
+        } else if ( 'count' in context.options[0] ) {
+          return Object.assign({}, context.options[0], { considerComments: false });
+        } else {
+          return Object.assign({}, context.options[0], { count: 1 });
+        }
+      } 
+      return options = { count: 1, considerComments: false };
```

Comment:
let’s avoid conflating assignment with an expression; it probably shouldn’t return anything if it’s meant to have side effects.

Reply:
Yep, corrected it in the latest commit.